docs/src/equations.md

```diff
@@ -375,3 +375,17 @@ It is assummed that some fraction ``\alpha`` of snow is melted during the proces
 ```math
 \frac{d}{dt} \rho e = \rho \mathcal{S}_{acc} ((1+\alpha) I_{liq} - \alpha I_{ice} + \Phi)
 ```
+
+### Stability and positivity
+
+Each source terms are individually limited such that they don't exceed the
+  available tracer specific humidity.
+```math
+\mathcal{S}_{x \rightarrow y} = min(\mathcal{S}_{x \rightarrow y}, \frac{q_{x}}{dt})
+```
+This will not ensure positivity because the sum of all source terms,
+  combined with the advection tendency,
+  could still drive the solution to negative numbers.
+It should however help mitigate some of the problems.
+The source terms functions treat negative specific humidities as zeros,
```

Comment:
Does this mean we're not conserving water in this case? 🤔

Reply:
I don't think so. The negative numbers are only treated as zeros when computing the tendencies for each process. We don't actually overwrite anything in the state vector in Atmos.

For example if I have small negative rain specific humidity in one of my grid cells, I will still advect it around and keep the negative value in `Y` unchanged. But I will not compute the evaporation or collision tendency for that grid cell. This may lead to errors in the microphysics representation, but not to the conservation budget for mass/water etc.

Reply:
That makes sense!